packages/components/src/ProgressBar/_docs/ProgressBar.stories.tsx

```diff
@@ -44,6 +47,46 @@ export const Playground: Story = {
   },
 }
 
+const colors = [
+  {
+    title: "Blue",
+    props: { color: "blue" },
+  },
+  {
+    title: "Green",
+    props: { color: "green" },
+  },
+  {
+    title: "Red",
+    props: { color: "red" },
+  },
+  {
+    title: "Yellow",
+    props: { color: "yellow" },
+  },
+] satisfies Array<{ title: string; props: Partial<ProgressBarProps> }>
+
+export const Colors: Story = {
+  render: () => (
+    <div className="flex flex-col gap-16">
```

Comment:
Nit: Put this in a decorator - the code snippet will look a little cleaner

Reply:
Not worthhhh ;P